anyway/parsers/infographics_data_cache_updater.py

```diff
@@ -117,7 +117,7 @@ def get_cache_info():
 
 def main(update, info):
     if update:
-        logging.info("Refreshing infographics cache...")
+        logging.info("Refreshing infographics cache...") 
```

Comment:
by mistake?

Reply:
@idosavion is right :) I think that this is a good idea, but there are files, that you can't use this method: 

### anyway/static/data/news_flash_infographics_mock/mock_data.json   
contains words in utf-16
### anyway/tests/accidents_definitional_ynet.tsv:
I'm not sure if there are constant in a tsv file.